code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -10,8 +10,8 @@
 	layer = 2.9
 	density = TRUE
 	anchored = TRUE
-	idle_power_consumption = 5
-	active_power_consumption = 100
+	idle_power_consumption = 0
+	active_power_consumption = 0
```

Comment:
Remember, it's important to test your PRs before submitting them!
You need to set ``requires_power`` to FALSE, this will just remove it's power drain vs actually letting it work without power
you also applied this to the basetype smartfridge, not the subtype